src/Task/ApiGen/ApiGen.php

```diff
@@ -48,6 +59,31 @@ public function __construct($pathToApiGen = null)
     }
 
     /**
+     * Pass methods parameters as arguments to executable. Argument values
+     * are automatically escaped.
+     *
+     * @param string|string[] $args
+     *
+     * @return $this
+     */
+    public function args($args)
+    {
+        if (!is_array($args)) {
+            $args = func_get_args();
+        }
+        $args = array_map(function ($arg) {
+            if (preg_match('/^\w+$/', trim($arg)) === 1) {
```

Comment:
I see. This is an override. It might be clearer if you called `parent::` at some point, if possible.

Backwards compatibility trumps API design (until we're ready for a new major release), so forget about what I said about an `operation()` method if doing that would introduce ambiguities.

If doing things a certain way to maintain backwards compatibility, please add comments explaining why the code is like this, and what the preferred method is.